### PR TITLE
[AWS] Move VPN configuration file into integrations and add tag collection

### PR DESCRIPTION
**Pull request by kaiyan-sheng**
## What does this PR do?

This PR is to move VPN configuration file into integrations repo and also added the config for collecting tags.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### CODE REVIEW

File: packages/aws/data_stream/vpn/agent/stream/stream.yml.hbs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-metricsets: ["vpn"]
+metricsets: ["cloudwatch"]
```
Comment (zmoog) [inline packages/aws/data_stream/vpn/agent/stream/stream.yml.hbs:1]: Just to test my understanding about how things work: this metricset name is matching the defined at https://github.com/elastic/beats/blob/main/x-pack/metricbeat/module/aws/cloudwatch/cloudwatch.go#L27, right?

Comment (zmoog) [reply packages/aws/data_stream/vpn/agent/stream/stream.yml.hbs:1]: And the tags collection part is handled is handled by the cloudwatch input type, right?

**Review by zmoog — APPROVED** [review]

File: packages/aws/data_stream/vpn/agent/stream/stream.yml.hbs (hunk 1 above)
Comment (kaiyan-sheng) [reply packages/aws/data_stream/vpn/agent/stream/stream.yml.hbs:1]: Yes, this is to move the config files (using cloudwatch as lightweight module) from Beats repo to Integrations and at the same time adding tag collection.

**Comment by kaiyan-sheng** [comment]
> Oh, I missed the changelog anche the package version, next time I will pay more attention! 😅

@zmoog Hey not your fault haha I did it on purpose so I don't have to fix merge conflict 😂

**Pull request merged**